package.json

```diff
@@ -24,13 +24,8 @@
     "publish-beta": "bash ./tools/deploy.sh stable beta",
     "scenes:generate": "ng e2e --port 4201 scenes && yarn scenes:optimize",
     "scenes:optimize": "node tools/optimize-scene-screenshots",
-    "test:audit:a11y": "node tools/audit-docs-a11y",
-    "test:audit:a11y:localhost": "run-p --race \"~~light-server -s dist/material-angular-io -p 4200 --quiet\" \"test:audit:a11y http://localhost:4200\" --",
-    "test:audit:a11y:ci": "run-p --race \"~~light-server -s ../dist/material-angular-io -p 4200 --quiet\" \"test:audit:a11y http://localhost:4200\" --",
-    "test:audit": "node tools/audit-docs",
-    "test:audit:localhost": "run-p --race \"start:static\" \"test:audit http://localhost:4200 2000\" --",
```

Comment:
Note this was previously broken because there is no `start:static`. If i switch it to the same as the others using `dist/material-angular-io` then I get the same failure locally (before any changes from this PR) as on CI in this PR.